catalog/spatial/registry/registry-source-configuration-handler/src/main/java/org/codice/ddf/registry/source/configuration/SourceConfigurationHandler.java

```diff
@@ -297,15 +315,18 @@ private Configuration findOrCreateConfig(String factoryPid,
      * @throws IOException
      */
     private void activateDeactivateExistingConfiguration(String fPidToActivate,
-            Map<String, Configuration> fpidToConfigurationMap) throws IOException {
+            Map<String, Configuration> fpidToConfigurationMap, String bindingType) throws IOException {
         String fpid;
+        String fpidToActivateWithBinding = fPidToActivate + bindingType;
+        String disabledFpidToActivateWithBinding =
+                fPidToActivate.concat(DISABLED_CONFIGURATION_SUFFIX) + bindingType;
```

Comment:
Since you are already using `concat()` you might as well use it again for `bindingType` or use `String.format`